pkg/api/frontend_logging.go

```diff
@@ -56,6 +68,9 @@ func (event *frontendSentryEvent) ToLogContext() log15.Ctx {
 	if event.Exception != nil {
 		ctx["stacktrace"] = event.Exception.FmtStacktraces()
 	}
+	if event.Contexts != nil {
+		addEventContextToLogContext("context", &ctx, &event.Contexts)
+	}
```

Comment:
There's no need to check if `event.Contexts` is nil because iterating over a nil map isn't an error, it behaves the same as if it were empty (you can read about this [here](https://blog.golang.org/maps)):
```suggestion
	addEventContextToLogContext("context", ctx, event.Contexts)
```

Reply:
Oh boy, go has some unconventional design decisions :) though does makes sense. Fixed-